### ISSUE TITLE
Hi，an issue about th hyper-parameter alpha

### ISSUE THREAD

**Issue by tbontbiaq** [gusye1234/LightGCN-PyTorch#23]
![image](https://user-images.githubusercontent.com/53202794/159485706-d822b726-4be6-4b94-9c38-bd03325e577a.png)
in
Hi，in the paper:LightGCN which is published in SIGIR ,2020.The formula 4 means k layer embedding should  multiple a weight 1/(k+1),but in your code ( model.py ,function computer), different layer embedding share the same weight.Is there anything wrong?
`        for layer in range(self.n_layers):
            if self.A_split:
                temp_emb = []
                for f in range(len(g_droped)):
                    temp_emb.append(torch.sparse.mm(g_droped[f], all_emb))
                side_emb = torch.cat(temp_emb, dim=0)
                all_emb = side_emb
            else:
                all_emb = torch.sparse.mm(g_droped, all_emb)
            embs.append(all_emb)
        embs = torch.stack(embs, dim=1)
        #print(embs.size())
        light_out = torch.mean(embs, dim=1)
        users, items = torch.split(light_out, [self.num_users, self.num_items])`

您好：论文里公式4，第k层的embedding是需要乘一个系数1/(k+1)再加和。但代码model.py的computer函数里，是直接将若干层的embedding取均值。请问这是什么情况？

**Comment by Yukayo** [comment]
他这个1/k+1的系数不就是所有人的平均么
